src/test/java/com/google/sps/data/friend_map/FriendsMapCreationTest.java

```diff
@@ -29,6 +29,7 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableMap;
```

Comment:
I'm having trouble figuring out how to add a comment in the right place - apparently I shouldn't have put this off :)

Your set up method isn't usually how this works. Having some statics at the top to use is a good idea. Normally you then construct the specifics for a test case within the test case itself. This keeps the scenario right in front of the reader instead of having to look back up to somewhere else. This works with good static variable names so it's clear what they are too.

Reply:
Ok! Since most of what is currently in the setUp method could just be constructed within the test cases that they apply to, do I still need to keep the setUp method here?

Reply:
No, you likely don't need it at all for a test class like this.

Reply:
Ok, got it!